src/nwb_datajoint/common/common_ephys.py

```diff
@@ -74,6 +74,11 @@ def make(self, key):
                         break
             if 'probe_type' not in key:
                 key['probe_type'] = 'unknown-probe-type'
+            # Define target_hemisphere based on targeted x coordinate
+            if electrode_group.targeted_x >= 0:  # if positive or zero x coordinate
+                key["target_hemisphere"] = "right"  # define target location as right hemisphere
```

Comment:
`target_hemisphere` is an enum and must be 'Right' or 'Left', i.e. must be capitalized

Reply:
Noted.